### DISCUSSION TITLE
Allow sharing with multiple contacts at once

### DISCUSSION THREAD

**Discussion by vishnukvmd** [ente-io/ente#824] [Enhancements]
Currently the checkbox works like a radio button. We should let users select multiple contacts on this screen.

<img src="https://github.com/ente-io/photos-app/assets/1161789/af7ec58c-382f-4650-b7df-9229ae48f835" width=420/>

**Comment by ashilkn** [comment]
Feature is ready.
https://github.com/ente-io/ente/pull/1116

Closing the discussion.
